kura/distrib/src/main/ant/build_equinox_distrib.xml

```diff
@@ -995,6 +996,8 @@ Resource-Processor: org.eclipse.kura.deployment.customizer.upgrade.rp.UpgradeScr
 				value=", reference:file:${kura.install.dir}/kura/plugins/jdk.dio_${jdk.dio.version}.jar@3" /> 
 			<entry key="osgi.bundles" operation="+"
 				value=", reference:file:${kura.install.dir}/kura/plugins/org.apache.commons.net_${org.apache.commons.net.version}.jar@3:start" />
+			<entry key="osgi.bundles" operation="+"
+                value=", reference:file:${kura.install.dir}/kura/plugins/org.apache.commons.exec_${org.apache.commons.exec.version}.jar@2:start" />
```

Comment:
If should be available early. As the modem lookup is early as well. However I guess the "start" keyword could be omitted altogether since there is (AFAIK) no activator present in the bundle.

Then again most bundles in Kura are "started" anyway.